### ISSUE TITLE
this test is 75% redundant and only measures the scale of conservative vs liberal

### ISSUE THREAD

**Issue by milahu** [8values/8values.github.io#116]
redundancy is a problem with many tests, like MBTI or big-five

in my understanding, all the 8value questions only measure **one single scale**:

conservative politics | liberal politics
------------ | -------------
sedentary culture, tribalism? | nomadic culture
military government [fire] | weak government [water]
weak citizens [earth] | military citizens [air]
guns are privilege [fire] | guns for all [air]
drugs are privilege [fire] | drugs for all [air]
regional tourism | global tourism
local self-sufficiency, grow your own, eat regional+seasonal food | global trade, import exotic food from far away
anti migration | pro migration
breed to stabilize one phenotype or "race", well-adapted to the local climate | breed to hybridize, inter-racial breeding
stability | variety
quality | diversity
primitive culture, low risk, high quality | high technology, high risk, high diversity
natural medicine | synthetic medicine, genetic engineering, transhumanism
traditional education [earth] | progressive education [air]
minority vote [fire] | majority vote [air]
majority makes poor decisions [earth] | minority makes poor decisions [water]
accumulate wealth, save money | distribute wealth, spend money
sexual monogamy [fire] | sexual liberty, bisexual [air]
mesomorph | endomorph + ectomorph = extramorph
many strong small states | few weak large states
competition of states | union of states
two class society, local order | classless society, local chaos
religious, monotheism, one god [fire] | atheist, polytheism, many gods [air]
low tax | high tax, eat the rich
private schools | public schools
private insurance | public insurance
exchange economy, market | gift economy, steal and donate
some humans are better | all humans are equal
some cultures are better | all cultures are equal
practical | rational [carl jung]
concrete | abstract [david keirsey]
cycloThymia | schizoThymia [Ernst Kretschmer]
vagotonia | sympatheticotonia
rest + digest | fight + flight
depressed | manic

in terms of [Milton Rokeach](https://en.wikipedia.org/wiki/Political_spectrum#Milton_Rokeach) and the [classical four elements](https://en.wikipedia.org/wiki/Classical_element) and Hans Eysenck two-factor terminology (extraversion + psychoticism)

conservative politics | liberal politics
------------ | -------------
communism + capitalism | socialism + nazism
fire + earth | air + water
satan + belial | lucifer + leviathan
equality-authority + difference-freedom | equality-freedom + difference-authority
extravert-psychotic + introvert-neurotic | extravert-neurotic + introvert-psychotic
young-masculine + old-feminine | young-feminine + old-masculine
son + mother | daughter + father
matriarchy [earth = mother] | patriarchy [water = father]

in a [two-factor-model](https://en.wikipedia.org/wiki/Two-factor_models_of_personality) with the two orthogonal factors equality and freedom, the two conservative types [fire + earth] share the **same diagonal**, and the two liberal types [air + water] share the other diagonal.

x | young | old
------------ | ------------ | -------------
**masculine** | fire | water
**feminine** | air | earth

in theory, there are five types of questions, five scales to measure:

1. conservative vs liberal = fire+earth vs air+water = diagonal, angle
2. inner age = young vs old = fire+air vs earth+water
3. inner gender = feminine vs masculine = earth+air vs fire+water
4. fire-ness = fire vs air+water vs earth, could be called "dominance"
5. air-ness = william sheldon scale = endo vs meso vs ecto, could be called "optimism"

the current 8values test pretends to give four scales, but really gives just one scale.

**Comment by ten7ei** [comment]
This is just some theory that puts everything into one scale. You can also put all questions in yin and yang for example. But you can get for example more information if you make additional axis on authority, because you put communism on the conservative site which is only true when you want authoritarian government. At the same time there are people who are liberal but are opposed to technologies like people for environmental protection.
All 8 scales give additional information where you can be extreme in one scale without being in your scale extreme.

**Comment by milahu** [comment]
> you can get for example more information if you make additional axis

yes, but the axis must be orthogonal.

my claim is, the four axis ( econ / dipl / govt / scty ) are much too similar,
and do not add much/any additional info

![four similar dimensions vs four orthogonal dimensions](https://user-images.githubusercontent.com/12958815/85059523-1303f200-b1a4-11ea-9b7e-c993e3ca7db4.png)

my solution to "four orthogonal dimensions" is

1. inner age
2. inner gender
3. outer age
4. outer gender

"( econ / dipl / govt / scty ) are much too similar"
more precise,
the axis pretend to be linearly independent [orthogonal],
but the **questions** do not reflect that.
[ to make this test better, we need better questions. ]

just have a look at the weight-factors for ( dipl / govt / scty ).
for all questions, they have the **same sign** ( positive / negative ),
with only one exception:

```js
{
    "question": "Even when protesting an authoritarian government, violence is not acceptable.",
    "effect": {
        "econ": 0,
        "dipl": 5,
        "govt": -5,
        "scty": 0
    }
},
```

.... which is a bug in my view, both factors should be negative.

conservatives only do peaceful protests, or passive protest like labor strikes.
liberals can have violent insurgencies / revolutions.
in conservative culture, violence always is "from above", from police or military.
in liberal culture, the majority of citizens have guns, and the weak government is under pressure.

> This is just some theory

this whole test in its current version is just a wild theory,
from somebody who is obviously untrained in the art of factor analysis.

when you [study the MBTI test](https://www.reddit.com/r/mbti/comments/ds3r3o/decoding_mbti_four_elements_alchemy_physiognomy/), for example, 50% is redundant

the questions only appear to be different, cos they use different words and phrases,
but if you happen to master the art of translation and **abstraction**,
[ im afraid, only fire is good at this, so only a minority will truly understand ]
you easily notice the strong similarity.

outer values | hans eysenck | MBTI 1+2 | MBTI 3+4
-- | -- | -- | --
young age | extravert | iNtuition | Perceiving
old age | introvert | Sensing | Judging
feminine gender | neurotic | Extraversion | Feeling
masculine gender | psychotic | Introversion | Thinking

the **only** benefit of redundant dimensions is,
you can measure how well the test questions were understood.
when similar questions are answered consistent, understanding was good.
when similar questions bring opposite answers, understanding was bad.

understanding seems to be more bad for "old age" types, earth and water.
these are anti-sensitive, have poor self-value, tend to have false self-images

**Comment by ten7ei** [comment]
Thank you for providing all these details. I'm not sure if I misunderstood some things but to me it looks like that you want to make a personality quiz.

I think this quiz is meant to be a political quiz. Maybe one can argue that political values are reflected from the personality but it doesn't mean that people with the same personality can have very different political values. 

So maybe you want to say that this quiz is mostly redundant if you want to measure the personality of someone. But for measuring the political values this is not the case.

Which political values do you think are less redundant than the 8 chosen ones? Or do you think most politics can only be devided into conservative VS. liberal?

**Comment by Genora51** [comment]
I'd like to add here that there is some interesting critique of some redundancy in the questions. However, each of the axis has a number of questions which only affect that axis, so it is certainly possible to achieve a unique result on each axis. In other words, although there are questions which affect more than one axis (and which have a tendency to affect those axes similarly across the quiz) there are enough single-axis-affecting questions to allow unique results.

It's also worth noting that the entire reason 8values exists is as a counter to the reductive notion that politics is a single axis, and to provide additional nuance to the often of the left and right wing that other political tests struggle to capture (see political compass).

That said, there is definitely merit in the factor analysis @milahu has done here, and if you can suggest how the test might be improved to reduce redundancy and increase linear independence in the 4 axes, I am all ears and I would love to hear those suggestions.

**Comment by Arakkun** [comment]
Actually, that's why having a dataset would be useful.
Is the model simplifiable? Doing a PCA or other dimensionality reductions of the results ( or even on the questions themselves ) Would really help finding that

**Comment by nissubjtsdjarwylwz** [comment]
The test is clearly only appicable to US, as progressive vs conservative are terms subjective to country's tradition and can contain different ideologies in different countries.

**Comment by Genora51** [comment]
Certainly, the test is written from and targeted at a western perspective; however, I disagree that it is not applicable elsewhere than the US; I for one am not from the States and yet the 8values test is perfectly applicable to my politics and the political spheres of the country I reside in.
